daprdocs/content/en/developing-applications/building-blocks/actors/actors-overview.md

```diff
@@ -102,3 +102,6 @@ The following example illustrates the above concepts. Consider an actor type tha
 
 <img src="/images/actors_background_concurrency.png" width=600>
 
+#### Scaling reminders
```

Comment:
Since we don't have a reminder section here not sure we need one for scaling them. This might be better located in https://docs.dapr.io/developing-applications/building-blocks/actors/howto-actors/#actor-timers-and-reminders (unless we combine them as I stated in another comment)

Reply:
The page you are referring to is modified in the next file in this PR. Is the action item to remove this section?

Reply:
The page you are referring to is modified in the next file in this PR. Is the action item to remove this section?

Reply:
Ideally it would be to add a reminders section here but since we will be refactoring all this I think for this PR, yes, remove it from here for now.